<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionShowAboutUsInfo"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Режим</string>
    </property>
    <widget class="QMenu" name="menuText">
     <property name="title">
      <string>Текст</string>
     </property>
     <addaction name="actionSetTextSendMode"/>
     <addaction name="actionSetTextRecieveMode"/>
    </widget>
    <addaction name="menuText"/>
   </widget>
   <addaction name="menuMode"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetTextSendMode">
   <property name="text">
    <string>Отправка</string>
   </property>
  </action>
  <action name="actionSetTextRecieveMode">
   <property name="text">
    <string>Приём</string>
   </property>
  </action>
  <action name="actionShowAboutUsInfo">
   <property name="text">
    <string>О нас...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
